sql/2025/performance/lcp_async_fetchpriority.sql

```diff
@@ -0,0 +1,109 @@
+CREATE TEMP FUNCTION getLoadingAttr(attributes STRING) RETURNS STRING LANGUAGE js AS '''
```

Comment:
Change all these functions to take JSON rather than STRING to save converting to STRING and back to JSON.